### ISSUE TITLE
insufficient color contrast in dark mode

### ISSUE THREAD

**Issue by Trott** [nodejs/nodejs.org#4196]
* **URL**: https://nodejs.org/en/blog/release/v17.0.0/
* **Browser version**: Chrome 94
* **Operating system**: macOS Catalina

See the byline and other text:

![image](https://user-images.githubusercontent.com/718899/137951618-36e30a74-3574-40f3-a389-67258334b506.png)

**Comment by bnb** [comment]
was literally about to post this exact same thing.

The byline is a span and the subheading is an h4. I assume h4-h6 are all too dark.
